modules/caddyhttp/encode/encode.go

```diff
@@ -112,7 +112,9 @@ func (enc *Encode) Provision(ctx caddy.Context) error {
 					"application/x-ttf*",
 					"application/xhtml+xml*",
 					"application/xml*",
-					"font/*",
+					"font/ttf*",
+					"font/otf*",
+					"font/x-woff*",
```

Comment:
This line isn't needed, WOFF is pre-compressed

Reply:
According to https://community.cloudflare.com/t/cloudflare-is-missing-the-current-woff-media-type-in-the-default-compression-config/679095, this is beneficial if Brotli (and I guess Zstandard) are used.

Brotli isn't natively supported by Caddy, but FrankenPHP and https://github.com/dunglas/caddy-cbrotli allow us to easily add support for it. Zstandard is supported and is now on by default for Chrome and Firefox (not for Safari).

Reply:
Fair enough, but if WOFF1 is to be compressed then the matcher should be the standardized `font/woff` instead of or in addition to miscellaneous names it went by before being standardized.

Reply:
TBH, I just copied the list provided by Cloudflare: https://developers.cloudflare.com/speed/optimization/content/brotli/content-compression/#compression-between-cloudflare-and-website-visitors

Reply:
Also won't that wildcard on the end cause it to match WOFF2 anyway? AFAICT that wildcard is there in case the server appends a charset to the content type, but these are binary formats so that shouldn't happen.

Reply:
The MIME type of WOFF2 `font/woff2`, so it will not match, but I can remove this MIME type if you want.

Reply:
WOFF removed

Reply:
Thanks; hopefully that's the best decision, but I'm not really sure, but it sounds like there is no real consensus anyway. Both WOFF versions are compressed, even if WOFF2 has "better" compression; but I don't see a need to compress either of them.

Appreciate it!